src/Nethermind/Nethermind.Runner/Program.cs

```diff
@@ -256,29 +251,29 @@ private static void Run(string[] args)
         }
     }
 
-    private static IEnumerable<ReadOnlyMemory<char>> GetDuplicateArguments(string[] args)
+    private static ReadOnlyMemory<char> GetArgumentName(string arg) => arg.StartsWith("--") ? arg.AsMemory(2) : arg.StartsWith('-') ? arg.AsMemory(1) : ReadOnlyMemory<char>.Empty;
+    private static IEnumerable<ReadOnlyMemory<char>> GetArgumentNames(IEnumerable<string> args)
```

Comment:
Make those methods local in `ValidateArguments`, they don't make sense outside of it.

Reply:
My main concern when I extracted those was `GetDuplicateArguments` not being able to reach that code.
I should have tested it before.
Thanks
Done